Missing ehtereumjs-vm dependency

**Describe the bug**
Direct installation of `smock` thats not work out of the box. When running my tests an error `Error: Cannot find module '@nomiclabs/ethereumjs-vm/dist/exceptions'` is thrown.
This is fixable by installing that dependency on the project, but its a pretty weird behaviour, honestly.

**Reproduction steps**

1. Start a project
2. Install smock
3. Try to use it (at least Im using `fake`)

**Expected behavior**
Work out of the box.

**Screenshots**
Can't show without doxing myself - approach via DM if necessary. 

**System Specs:**

- OS: Ubuntu
- Package Version (or commit hash): 2.0.3

**Additional context**
n/a

--- a/package.json
+++ b/package.json
@@ -52,6 +52,7 @@
     "@ethereumjs/vm": "5.3.1"
   },
   "dependencies": {
+    "@nomiclabs/ethereumjs-vm": "^4.2.2",
     "diff": "^5.0.0",
     "lodash.isequal": "^4.5.0",
     "lodash.isequalwith": "^4.4.0",
@@ -61,7 +62,6 @@
   "devDependencies": {
     "@commitlint/cli": "12.1.4",
     "@commitlint/config-conventional": "12.1.4",
-    "@nomiclabs/ethereumjs-vm": "^4.2.2",
     "@nomiclabs/hardhat-ethers": "2.0.2",
     "@nomiclabs/hardhat-waffle": "2.0.1",
     "@typechain/ethers-v5": "7.0.1",
@@ -100,7 +100,6 @@
     "@ethersproject/abi": "^5",
     "@ethersproject/abstract-provider": "^5",
     "@ethersproject/abstract-signer": "^5",
-    "@nomiclabs/ethereumjs-vm": "^4",
     "@nomiclabs/hardhat-ethers": "^2",
     "ethers": "^5",
     "hardhat": "^2"

